GameData/masTerTorch/Localization/en-us.cfg

```diff
@@ -0,0 +1,104 @@
+Localization
+{
+	en-us
+	{
+		// KDex
+	#KDEX_title = Kerbal Dust Experiment (KDex)
+	#KDEX_manufacturer = masTerTorch R&D Center
+	#KDEX_description = Inside this tiny coffee can is a highly sensitive sensor which can measure the density of dust particles colliding with the back of the can. Now we can extract science out of collisions!
+	#KDEX_tags = experiment dust collision research rerun re-run reus re-us redo re-do reset re-set science sensor kdex
+
+	#KDex_startEventGUIName = KDex: Lid Open 
```

Comment:
Why the change from KDEX_ to KDex_ as the prefix?

And: is the localisation system case-sensitive in either case?

Reply:
oversight. yes, case sensitive. kindly choose one, but as you probably know, change universally.

If being correct: probably KDEx would be correct. old keyboard probably messed up from old fingers.

Reply:
Actually I'd prefer KDEX_ as the prefix.